tests/ui/conftest.py

```diff
@@ -9,11 +9,21 @@
 from django.core.management import call_command
 from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
 from olympia.amo.tests import create_switch
+from selenium.webdriver.common.action_chains import ActionChains
+
 from pytest_django import live_server_helper
 
 
+@pytest.fixture
+def selenium(selenium):
+    selenium.implicitly_wait(10)
+    selenium.set_window_size(1920, 1080)
+    ActionChains(selenium).reset_actions()
+    return selenium
+
+
 @pytest.fixture(scope='function')
-def my_base_url(base_url, request):
+def my_base_url(base_url, request, live_server):
```

Comment:
This means running against a remote server will attempt to start up a live server instance, which we don't want.